src/website/app/LiveProvider.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import dedent from 'dedent';
+import {
+  LiveProvider as ReactLiveProvider,
+  LiveEditor,
+  LiveError,
+  LivePreview
+} from 'react-live';
+import { createStyledComponent } from '../../utils';
+
+type Props = {
+  backgroundColor?: string,
+  hideSource?: boolean,
+  isChromeless?: boolean,
```

Comment:
Maybe this should be "chromeless" rather than "isChromeless"?  We seem to have been reserving the "isNNN" convention for state related booleans and using raw "NNN" for props related booleans.

Reply:
Addressed.